app/navigation/nonsipp/UnregulatedOrConnectedBondsNavigator.scala

```diff
@@ -28,7 +28,7 @@ object UnregulatedOrConnectedBondsNavigator extends JourneyNavigator {
   override def normalRoutes: UserAnswers => PartialFunction[Page, Call] = userAnswers => {
     case page @ UnregulatedOrConnectedBondsHeldPage(srn) =>
       if (userAnswers.get(page).contains(true)) {
-        routes.UnauthorisedController.onPageLoad()
+        controllers.nonsipp.otherassetsheld.routes.OtherAssetsHeldController.onPageLoad(srn, NormalMode)
```

Comment:
The yes answer shouldn't go here, if the next page isn't built then go to unauthorised